src/engine/BMInterface.php

```diff
@@ -432,6 +432,15 @@ public function save_join_game_decision($playerId, $gameId, $decision) {
         $game = $this->load_game($gameId);
 
         if (BMGameState::CHOOSE_JOIN_GAME != $game->gameState) {
+            if (('reject' == $decision) &&
+                ($playerId == $game->playerIdArray[0])) {
+                $decision = 'cancel';
+            }
+            $this->set_message(
+                'Your decision to ' .
+                $decision .
+                ' the game failed because the game has been updated in the meanwhile'
```

Comment:
Sure, "meantime" is more grammatical than "meanwhile", but i still think it makes players ask "in the meantime since what?", and we might as well tell them what actually happened instead.  But, again, i don't feel at all strongly - i probably would have gone ahead and merged this except that i'd like shadowshade to add the responderTest